Gemfile

```diff
@@ -27,3 +27,5 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
+# Table of Contents Gem
+gem 'kramdown'
```

Comment:
Looks like it's still here though?

Reply:
Yes, because I removed kramdown it from the gemfile in commit 57d0710 3 days ago. 
When you said to revert my changes to the gemfile, I thought that this meant that I should revert change 57d0710, which I did with commit 5da641d yesterday. This added kramdown back in.

So... I'm still learning ruby on rails, and have read up and am confused. Do we need kramdown in the gemfile? It worked on my computer during the commit 3 days ago, but that might be because of some local connections that were previously created that I don't know how to do. 

If you don't know, but have ideas about what terms I could search to figure it out, that would be just as great as advice!

Reply:
As an aside, from my current understanding of gemfiles, they work like python packages, so I thought I need to include the ones I want to use functions from in the Gemfile, unless the function is inbuilt into the ruby on rails installation.

Reply:
This is still unnecessary. Does it not work without it on your machine? Jekyll includes many gems including kramdown. Check out the Gemfile.lock.

Reply:
It's fixed/been removed again, as of the accidentally misnamed commit about_FAQ (	d37fc37
)